hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/OwnerTransformations.scala

```diff
@@ -16,6 +16,28 @@ object OwnerTransformations {
     val secondaryAddressOwnership = secondaryAddress.flatMap {
       if (_) rawRecord.getOptionalNumber("oc_address2_own") else None
     }
+    
+    // Parsing the oc_primary_residence_census_division
+    // format is: "Division <N>: <some-description>"
+    val censusDivisionPattern: Regex = "(Division \\d+:)"
+    
+    /**
+      * Extract the Division value from the full text string.
+      *
+      * @param divisionString the text string to be parsed
+      * @return an integer of the division number
+    **/
+    def getCensusDivision(divisionString: String): (integer) = {
```

Comment:
Invalid type annotation here. Once you fix it the body will need tweaking to return the right type (it's currently a string)